test/schema-test.js

```diff
@@ -1,9 +1,7 @@
 const schema = require('../lib/schema');
-const vows   = require('vows');
-const assert = require('assert');
+const { expect } = require('chai');
 
-
-const torrent = () => {
+const createTorrent = () => {
```

Comment:
this can be updated to use the `before` pattern in other test files, assigning the created torrent to a similar `torrent` variable that can then be referenced by each test, and then be modified for that test.

the reason it was done this way before, was because i was trying to do a similar pattern for vows, which has no `before`